src/workloads/execution/BackgroundValidateCmd.yml

```diff
@@ -0,0 +1,117 @@
+SchemaVersion: 2018-07-01
+Owner: "@mongodb/product-perf"
+
+Actors:
+- Name: InsertData
+  Type: Loader
+  Threads: 1
+  Phases:
+  - Repeat: 1
+    Database: &db test
+    Threads: 1
+    CollectionCount: 1
+    DocumentCount: 50000
+    BatchSize: 10000
+    Document:
+      x: &x {^RandomInt: {min: 0, max: 2147483647}}
+      y: &y {^RandomInt: {min: -1000, max: 1000}}
+      z: &z {^RandomInt: {distribution: geometric, p: 0.1}}
+      a: &a {^RandomString: {length: 128}}
+      b: &b {^RandomString: {length: 15}}
+      arr:
+      - *x
+      - *y
+      - *z
+      loc2d: &loc2d [{^RandomInt: {min: -180, max: 180}}, {^RandomInt: {min: -180, max: 180}}]
+      loc2dSphere: &loc2dSphere [{^RandomInt: {min: -180, max: 180}}, {^RandomInt: {min: -90, max: 90}}]
+    Indexes:
+    - keys: {x: 1}
+    - keys: {x: -1}
+      options: {expireAfterSeconds: 1000000}
+    - keys: {x: 1, y: 1}
+    - keys: {x: 1, y: 1, b: 1}
+    - keys: {arr: 1, z: 1}
+    - keys: {arr: 1, b : 1}
+    - keys: {a: "text"}
+    - keys: {a: 1}
+      options: {sparse: true}
+    - keys: {a: "hashed"}
+    - keys: {loc2d: "2d"}
+    - keys: {loc2dSphere: "2dsphere"}
+  - &Nop {Nop: true}
+  - *Nop
+  - *Nop
+
+- Name: CrudWithoutValidation
+  Type: CrudActor
+  Database: *db
+  Threads: 25
+  Phases:
+  - *Nop
+  - Duration: 5 minutes
+    Collection: Collection0
+    Operations:
+    - OperationName: bulkWrite
+      OperationCommand:
+        WriteOperations:
+        - WriteCommand: insertOne
+          Document: {y: {^RandomInt: {min: -1000, max: 1000}}}
+        - WriteCommand: updateOne
+          Filter: {y: {min: -1000, max: 1000}}
+          Update: {$set: {y: {min: -1000, max: 1000}}}
+        - WriteCommand: deleteOne
+          Filter: {y: {min: -1000, max: 1000}}
+    - OperationName: find
+      OperationCommand:
+        Filter: {y: {min: -1000, max: 1000}}
+  - *Nop
+  - *Nop
+    
+- Name: CrudWithValidation
+  Type: CrudActor
+  Database: *db
+  Threads: 25
+  Phases:
+  - *Nop
+  - *Nop
+  - *Nop
+  - Duration: 5 minutes
+    Collection: Collection0
+    Operations:
+    - OperationName: bulkWrite
+      OperationCommand:
+        WriteOperations:
+        - WriteCommand: insertOne
+          Document: {y: {^RandomInt: {min: -1000, max: 1000}}}
+        - WriteCommand: updateOne
+          Filter: {y: {min: -1000, max: 1000}}
+          Update: {$set: {y: {min: -1000, max: 1000}}}
+        - WriteCommand: deleteOne
+          Filter: {y: {min: -1000, max: 1000}}
+    - OperationName: find
+      OperationCommand:
+        Filter: {y: {min: -1000, max: 1000}}
+
+- Name: Validate
+  Type: RunCommand
+  Threads: 1
+  Phases:
+  - *Nop
+  - *Nop
+  - Repeat: 1
+    Database: admin
+    Operation:
+      OperationMetricsName: MaxValidateMBperSec
+      OperationName: RunCommand
+      OperationCommand:
+        setParameter: 1
+        maxValidateMBperSec: 1
+  - Duration: 5 minutes
```

Comment:
I think Genny will keep running validates in a loop. The last one will go beyond the 5 minutes, and will hold the phase open for all the actors. The other actors should close on their own when they hit their time limits. I want to make sure there's no impact on the other actors stats because of the phase being held open for the validation command. @guoyr do you know what happens in that case?

Reply:
Would "Blocking: None" have the same behavior here or would it kill the background op?

Reply:
This will hold the actor open and the last one will go beyond 5 minutes regardless of if you specify repeat or blocking: None. 

If you want validate to finish first, you can either set a repeat of 4, or set the other actors that run in this phase to blocking: none so they will continue to run as long as validate is running.

Re David's question, there's no impact on the metrics of other actors (unless they're buggy)

Reply:
Should be fine the way it is then as it won't affect the other actors metrics!